presto-main/src/test/java/com/facebook/presto/sql/TestExpressionOptimizer.java

```diff
@@ -57,11 +59,26 @@
 
 public class TestExpressionOptimizer
 {
+    private TypeRegistry typeManager;
+    private ExpressionOptimizer optimizer;
+
+    @BeforeClass
```

Comment:
if there nothing that need to be released, why `@BeforeClass` is needed?

Reply:
that is?